src/Spain/Mews.Fiscalizations.Spain/Converters/ModelToDtoConverter.cs

```diff
@@ -172,21 +172,33 @@ private PersonaFisicaJuridicaType Convert(CounterParty counterParty)
                 },
                 foreign => new PersonaFisicaJuridicaType
                 {
-                    NombreRazon = foreign.Name,
+                    NombreRazon = foreign.Match(
+                        customer => customer.Name,
+                        company => company.Name
+                    ).Value,
                     Item = Convert(foreign)
                 }
             );
         }
 
         private IDOtroType Convert(ForeignCounterParty foreignCounterParty)
         {
+            var idType = foreignCounterParty.Match(
+                customer => customer.IdentificatorType,
+                company => company.IdentificatorType
+            );
             return new IDOtroType
             {
-                CodigoPais = Convert(foreignCounterParty.Country),
+                CodigoPais = Convert(foreignCounterParty.Match(
+                    customer => customer.Country,
+                    company => company.TaxpayerNumber.Country
+                )),
```

Comment:
maybe a var country might be a good idea as it's passed into the convert. But that's just an opinion, not why I'm returning this.